core/modules/menu/menu.module

```diff
@@ -392,34 +397,6 @@ function menu_parent_options_js() {
   return $options;
 }
 
-/**
- * Ajax callback to replace the menu item parent options.
- *
- * Whenever the item language changes, the available parents also have to get
- * updated.
- *
- * @see menu_edit_item().
- * @see menu_form_node_form_alter().
- */
-function menu_update_parent_options_ajax($form, $form_state) {
```

Comment:
I'm not sure, but this callback is used in both, menu_edit_item AND in menu_form_node_form_alter.

Removing it might beak the latter, as the change in this PR only addresses menu_edit_item(), it seems. Only if menu_form_node_form_alter() also switches (different issue), this callback gets obsolete.

Or am I missing something?

Reply:
Nope, it shouldn't be removed. Will restore it.